generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user-management.component.html

```diff
@@ -42,10 +42,12 @@
                     <div *ngFor="let authority of user.authorities">
                         <span class="tag tag-info">{{ authority }}</span>
                     </div>
-                </td><% if (databaseType == 'sql' || databaseType == 'mongodb') { %>
```

Comment:
This was to match the condition in the table header - the headers appear without the cells